contrib/palantir/src/mill/contrib/palantir/JavafmtModule.scala

```diff
@@ -0,0 +1,49 @@
+package mill
+package contrib.palantir
+
+import mill.util.Jvm
+
+trait JavafmtModule extends PalantirModule {
```

Comment:
Let's fold this and `PalantirModule` into a combined `JavaFormatModule`, since there isn't a term `javafmt` in common usage and just calling it `Palantir` is a bit too unspecific